db/migrations/temp_add_stripe_update_column.sql

```diff
@@ -0,0 +1,11 @@
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query - add column stripe_email_pending';
+ALTER TABLE users ADD COLUMN IF NOT EXISTS stripe_email_pending BOOLEAN NOT NULL DEFAULT FALSE;
```

Comment:
In case we are not sure if our current dataset is 100% synced, we could set it to true for stripe_customer_id users to sync everyone initially. Thoughts?

Reply:
If we want to do this I would set the flag manuely for all users AFTER we tested that the implementation actually works for test users.